### PR TITLE
VAN-385: add label for each input field

### PR DESCRIPTION
**Pull request by zainab-amir**
Ticket: https://openedx.atlassian.net/browse/VAN-385

Screenshot 2021-02-22 at 4 50 13 PM

### CODE REVIEW

File: src/_style.scss [hunk 1]
```diff
@@ -14,6 +14,22 @@ $microsoft-focus-black: #000;
...
+  height:1px;
+  overflow:hidden;
+}
+
+.focus-out {
+  position: absolute;
+  padding-left: 17px;
+  opacity: 0.75;
+  z-index: 1;
+}
```
Comment (adeelkhan) [inline src/_style.scss:31]: What this focus-out class needed for?

File: src/common-components/AuthnValidationFormGroup.jsx [hunk 2]
```diff
@@ -38,12 +35,18 @@ const AuthnCustomValidationFormGroup = (props) => {
...
-      </Form.Label>
-    ) : <span />;
-    return fieldLabel;
+    let className;
+    if (props.optionalFieldCheckbox || (!showLabelText && (props.value !== '' || props.type === 'select'))) {
+      className = 'sr-only';
+    } else if (showLabelText) {
+      className = 'pt-10 h6';
+    } else {
+      className = 'pt-10 focus-out';
```
Comment (adeelkhan) [inline src/common-components/AuthnValidationFormGroup.jsx:44]: And here?

File: src/_style.scss (hunk 1 above)
Comment (zainab-amir) [reply src/_style.scss:31]: I have mimicked the functionality of FE. 

- The placeholder is removed
- Label is used in place of placeholder i.e move the Label down above the form field using focus-out class
- Label is moved up when user focus in

Comment (adeelkhan) [reply src/_style.scss:31]: Got it! Can we add some unit tests if possible to verify .sr-only class is applied on required fields.

File: src/_style.scss [hunk 3]
```diff
@@ -14,6 +14,22 @@ $microsoft-focus-black: #000;
 $apple-black: #000000;
 $apple-focus-black: $apple-black;
 
+.sr-only {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
```
Comment (adeelkhan) [inline src/_style.scss:23]: Can you add space between key value pairs

```suggestion
  overflow: hidden;
```

**Review by adeelkhan — APPROVED** [review]

**Comment by adeelkhan** [comment]
Also the register page i am seeing is a bit different from yours. See the labels.
Screen Shot 2021-02-24 at 10 10 45 AM

**Comment by zainab-amir** [comment]
@adeelkhan The change is because of the PR that I merged yesterday to make text sentence cased. The screenshot I have above was taken before that.

**Pull request merged**